packages/client/src/components/SafeNFTs.js

```diff
@@ -73,7 +76,7 @@ function SafeNFTs({ web3, address }) {
                   sender={address}
                   initialState={{
                     assetType: ASSET_TYPES.NFT,
-                    selectedNFT: token,
+                    selectedNFT: { ...token, collectionName: collectionId },
```

Comment:
Token doesn't have information about a collection, so I added it here to make sending tokens from NFTs page work. When user opens the modal, the NFT will be selected, and there will be no errors when this NFT is sent to the account.